packages/manager/apps/ips/src/utils/ipRange.ts

```diff
@@ -0,0 +1,9 @@
+// Regular expression pattern for /24 subnet
+export const pattern = /^(?:(?:25[0-5]|(2[0-4]|1\d|[1-9]|)\d)(\.(?!$)|$)){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?\/((19|2[0-4]))?)$/;
+
+export const ipRangePlaceholder = '192.168.22.42/24';
+
+// The subnet address is limited to only "/24".
+export const isValidIpRange = (subnet: string): boolean => {
+  return new RegExp(pattern).test(subnet);
+};
```

Comment:
I think it's specific to byoip so it should be inside the byoip folder since it won't be reused by other pages